nohup.out

```diff
@@ -188,3 +188,52 @@ Harp v0.20.3 – Chloi Inc. 2012–2015
 Your server is listening at http://localhost:9000/
 Press Ctl+C to stop the server
 ------------
+
```

Comment:
Please be careful with what you commit, try `git commit <files>` instead of `git commit .`